CHANGELOG.md

```diff
@@ -31,11 +31,13 @@ This project adheres to [Semantic Versioning](https://semver.org/).
    * Added the ability to only download new files if remote file listing
      capabilities are not available for the Instrument.
    * Added kwargs for epoch units and origin in `pysat.utils.io.load_netCDF`.
-   * Vectorized `Meta.var_case_name` and `Meta.attr_case_name` to support 
+   * Vectorized `Meta.var_case_name` and `Meta.attr_case_name` to support
      list of str as input as well as str.
    * Added a time function to calculate decimal year from datetime.
    * Allow `Instrument.rename` to take a fuction or mapping dict as input,
      after adapting routine to use `Meta.rename`
+   * Added an update method and type evaluation method to MetaLabels.
+   * Allowed MetaLabels to be expanded through setting new Meta data values.
```

Comment:
```
inst.meta['test'] = {'units': 'yeah', 'new_label': 'old_info'}
```
works nicely!

Formatting is bad below, I also removed non-relevant variables.
```
In [6]: inst.meta
Out[6]: 
pysat.Meta(metadata=                 units            long_name                                              notes                                 desc value_min value_max fill new_label
        
test              yeah                 test                                                                                               NaN       NaN  NaN  old_info
```